tekton/pipeline_run_test.go

```diff
@@ -105,7 +105,12 @@ var _ = Describe("PipelineRun", func() {
 			},
 			Spec: ecapiv1alpha1.EnterpriseContractPolicySpec{
 				Description: "test-policy-description",
-				Sources:     []string{"foo"},
+				Sources: []ecapiv1alpha1.Source{
+					ecapiv1alpha1.Source{
+						Name:   "foo",
+						Policy: []string{"https://github.com"},
```

Comment:
I assume the actual value doesn't matter, but for readability, aybe a more realistic is better:

    Policy: []string{"https://github.com/company/policy"},

Reply:
There's also a "Data" key:

    Data: []string{"https://github.com/some-company/data"},

Reply:
Yep, that's fine. (I mentioned just as an fyi.)

Reply:
thanks. added Data